index.php

```diff
@@ -10,6 +10,10 @@
 <?php
 include 'includes/header.php';
 ?>
+
+  <!-- content -->
+
+
```

Comment:
@manishnetsol This is Test
@dheeth This is Test

Reply:
@pulkitnagpal007 This is done, please review again.